framework/src/media/MediaRecorderImpl.cpp

```diff
@@ -23,6 +23,8 @@
 #include "audio/audio_manager.h"
 
 namespace media {
+//ToDo: This handler needs to be moved into MediaWorker.cpp
+audio_handler_t* mInAudioHandler;
```

Comment:
I feel coding guidelines violated,
audio_handler_t* mInAudioHandler;
==>(change to)
audio_handler_t *mInAudioHandler;